datastore/src/main/java/io/spine/server/storage/datastore/Indexes.java

```diff
@@ -24,26 +24,30 @@
 import com.google.cloud.datastore.EntityQuery;
 import com.google.cloud.datastore.Key;
 import com.google.cloud.datastore.Query;
-import com.google.common.base.Function;
 import com.google.common.collect.Iterators;
+import com.google.common.collect.Streams;
+import io.spine.server.storage.Storage;
```

Comment:
I think this import statement is only used for `{@linkplain}`s and therefore the FQNs in the links should be restored and the import should be removed.